### PR TITLE
MAT-5395, MAT-5394: Add endorserSystemID, update to hardcoded value if endorsed

### PR DESCRIPTION
**Pull request by mcmcphillips**
## PR Template

Jira Ticket: [MAT-5395](https://jira.cms.gov/browse/MAT-5395)

Jira Ticket2: [MAT-5394](https://jira.cms.gov/browse/MAT-5394)
(Optional) Related Tickets:
https://github.com/MeasureAuthoringTool/madie-java-models/pull/103/files

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e. Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [x] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e. Changing auth methods, Adding a new user type, Adding a required but vulnerable package)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (adongare) [inline package.json]: can we delete the outer package.json and lock file? it uses the one inside

Comment (mcmcphillips) [reply package.json]: Yes, accidental addition, ty

**Review by adongare — APPROVED** [review]
